vendor/src/github.com/docker/swarmkit/manager/controlapi/service.go

```diff
@@ -18,6 +18,7 @@ import (
 
 var (
 	errNetworkUpdateNotSupported = errors.New("changing network in service is not supported")
+	errRenameNotSupported        = errors.New("service renaming is not supported")
```

Comment:
yes, that may be better (I based this one on "service mode change is not allowed" below)